lib/python/pyflyby/_comms.py

```diff
@@ -0,0 +1,79 @@
+from __future__ import absolute_import, division
```

Comment:
Please import `print_function` here too, to help keep things Python 3 compatible once https://github.com/pyflyby/pyflyby/pull/10 is finished.